openslides/urls.py

```diff
@@ -40,9 +35,13 @@
 )
 
 # TODO: move this patterns into core or the participant app
+js_info_dict = {'packages': []}
+
 urlpatterns += patterns(
     '',
-    (r'^jsi18n/$', 'django.views.i18n.javascript_catalog', js_info_dict),
+    url(r'^jsi18n/$',
+        'django.views.i18n.javascript_catalog',
+        js_info_dict),
```

Comment:
The js_info_dict can be removed right away. We definitely need another way to handle translation in angular